recipes/yaclib/all/conanfile.py

```diff
@@ -93,6 +93,10 @@ def validate(self):
                 self.output.warn(
                     "yaclib recipe does not recognize the compiler. yaclib requires a compiler supporting c++17. Assuming it does.")
 
+        # TODO: Fix me
+        if self.settings.compiler == "clang" and self.settings.compiler.libcxx == "libstdc++":
+            raise ConanInvalidConfiguration("Couldn't build yaclib wtih clang and libstdc++")
```

Comment:
What exactly error you have?
This configuration exist in yaclib CI

Reply:
This is the older pre gcc-5 ABI one if I am not mistaken ... I dont think we build this in CCI 🤔

Reply:
@prince-chrismc I think I saw this configuration in cci

@MBkkt this configuration builds fine, but then it links to the test_package executable can't find some standard definitions like ~exception_ptr and optional::optional
It's a shame for me not to find root cause of this problem.

Reply:
Interesting, exception_ptr is really used, but optional is not.
https://github.com/YACLib/YACLib/search?q=optional&type=code

Is it possible it was for old version?

Reply:
I don't know why and how, but this problem resolved itself
I have no idea how to reproduce it again :man_shrugging: